bx_py_utils/test_utils/snapshot.py

```diff
@@ -9,6 +9,23 @@ def _write_json(obj, snapshot_file):
         json.dump(obj, snapshot_handle, ensure_ascii=False, indent=4, sort_keys=True)
 
 
+def assert_text_snapshot(root_dir: Union[pathlib.Path, str], snapshot_name: str, got: str):
+    assert re.match(r'^[-_.a-zA-Z0-9]+$', snapshot_name), f'Invalid snapshot name {snapshot_name}'
+    assert isinstance(got, str)
+
+    snapshot_file = pathlib.Path(root_dir) / f'{snapshot_name}.snapshot.txt'
```

Comment:
The filename is incosistent with the file naming of the JSON snapshot (it has no `.snapshot` in it). It would be good to keep them consistent.
For my preference the JSON snapshot should also contain the `.snapshot` bit, this way the file name by itself tells what the file contains.

Reply:
Yes, the plan is to always include `.snapshot` in `assert_snapshot` too. Hence me sneakily including `.snapshot` in all new calls to `assert_snapshot`!